website/src/pages/index.tsx

```diff
@@ -24,7 +27,7 @@ function Index() {
         <Enterprise />
         <Quote />
         <Integrations />
-        <NewsletterSignup />
+        <NewsletterSignup hidden={i18n.currentLocale !== "en"} />
```

Comment:
@torotimes Do we want to hide newsletter signup for non-english speakers?

Reply:
I couldn't find any reference right now, so I might have misunderstood or remember wrong. But I think it was mentioned on several occasions that the newsletter form should only be on the English site. Correct me if I'm wrong, @torotimes

Reply:
Correct.